appSyncClient.go

```diff
@@ -159,10 +159,10 @@ func (client *appSyncClient) CreateOrUpdateResolvers(apiID string, resolversFile
 			})
 
 			if err != nil {
-				logger.Println(fmt.Sprintf("Function %s failed to create: %s", *function.Name, err))
+				logger.Println(fmt.Sprintf("Function %s failed to create: %s", function.Name, err))
 				functionStatistics.FailedToCreate++
 			} else {
-				functions = append(functions, function)
+				functions = append(functions, functionResponse)
```

Comment:
where the appended `functions` is used?

Reply:
After it creates/updates functions it begins work on resolvers. Resolvers can reference functions by their names. However that is not how AWS API expects them to be referenced.. they expect ARN. So `functions` is a lookup list to lookup those ARNs

Reply:
I was asking where it's used which is seems on line 193 in `getFunctionByName`